README.md

```diff
@@ -2,15 +2,56 @@
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Describe the functionality of arguments
+- Define methods that accept arguments
 
 ## Introduction
 
-## SWBAT 1
+In the real world, many interactions are customized or specific to our
+day-to-day experiences. For example, if you go to a coffee shop, they may take
+down your name to call up your order when it's ready. If visit a doctor's
+office, they may call your name when it's your turn to be seen. Variables that
+we experience in real life often need to be translated into to programs.
```

Comment:
I think you're 90% to the way of nailing this real world analogy. I think a sentence like this will seal it:

> In the example of the doctor's office or the coffee shop they have a "template" behavior that they apply to everyone, but that they change _only slightly_ depending on the particular customer or patient. Methods can also be written to have a "template" of shared activity but which can be changed just a little bit depending on some particular data. The _particular bits_ that make the change are called "arguments."

1. The focus of this piece is about arguments, show that arguments allow common template code + _just a bit of variance_
2. Make sure the r/w example hits home, _precisely_